accepted/2020/feature-switch.md

```diff
@@ -69,7 +69,7 @@ The property's value will be burned in by the linker when a corresponding featur
 ### Unified pattern for feature definition in MSBuild
 Introduce a standard pattern how to define these properties in the SDK. Basically a way to define a property and have it automatically passed through to `.runtimeconfig.json` as well as linker substitutions.
 
-This requires a mapping between the MSBuild property name, the full name of the feature switch for runtime configuration (which would show up in `.runtimeconfig.json`) and the read-only static property in the managed code which is used to branch the behavior. The mappings will be defined by [`RuntimeHostConfigurationOption`](https://github.com/dotnet/sdk/blob/36ef8b2aa8e5d579c921704bdab69a7407936889/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets#L347) which includes an AppContext configuration name and its value based on the values of user-facing MSBuild properties, and by `ILLink.Substitutions.xml` files embedded in the assembly defining the features. `RuntimeHostConfigurationOption` items will have optional boolean `Link` metadata which indicates that the feature setting should be optimized by the linker. This metadata can be set to `false` either in the SDK (for features which don't have defined linker optimizations) or by developers (if they want to disable linking of a particular feature). The default behavior will be for boolean `RuntimeHostConfigurationOption`s to be treated as if `Link == true`, using `ItemDefinitionGroup` or an appropriate MSBuild mechanism.
+This requires a mapping between the MSBuild property name, the full name of the feature switch for runtime configuration (which would show up in `.runtimeconfig.json`) and the read-only static property in the managed code which is used to branch the behavior. The mappings will be defined by [`RuntimeHostConfigurationOption`](https://github.com/dotnet/sdk/blob/36ef8b2aa8e5d579c921704bdab69a7407936889/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets#L347) which includes an AppContext configuration name and its value based on the values of user-facing MSBuild properties, and by `ILLink.Substitutions.xml` files embedded in the assembly defining the features. `RuntimeHostConfigurationOption` items will have optional boolean `Trim` metadata which indicates that the feature setting should be optimized by the linker. This metadata can be set either in the SDK (for features which have defined linker optimizations) or by developers (if they want to disable linking of a particular feature). Only items in `RuntimeHostConfigurationOption` with `Trim == true` will flow to the linker.
 
```

Comment:
We should add a line somewhere which says that `RuntimeHostConfigurationOption` item with `Trim == true` must have a boolean value (`true` or `false`) otherwise linker will fail on it.